README.rst

```diff
@@ -1,30 +1,40 @@
 |image0| |image1|
 
-A python util to fetch datasets from the LIBSVM website.
+A python util to fetch datasets from different databases.
+
+Currently supported databases are:
+
+- LIBSVM (libsvm_)
 
 
 Getting design matrix and target variable is as easy as:
 
 ::
 
-    from libsvmdata import fetch_libsvm
-    X, y = fetch_libsvm("news20.binary")
+   from libsvmdata import fetch_dataset
+   X, y = fetch_dataset("news20.binary")
 
+There is no need to specify the database name. Supported datasets can be displayed as:
+
+::
 
-Currently supported datasets are in ``libsvmdata.supported``.
```

Comment:
Open discussion: isn't it also nice to have them in a variable that can be used in the code?

Reply:
I think this should rather be a dictionary were keys are indicating the database (LIBSVM, Breheny, ...) ?